DiscordBot/commands/mud/move.js

```diff
@@ -0,0 +1,94 @@
+const commando = require('discord.js-commando');
+
+var entryRoomRole;
+var room0Role;
+var room1Role;
+var room2Role;
+
+class MoveCommand extends commando.Command {
```

Comment:
I would possibly add a piece to the constructor that manually goes through and makes an array of the rooms' IDs, that way things aren't hard coded.
This is fine for now though.

Reply:
I moved the room IDs to a separate file that can be referenced similarly to the loginCode.js. That way once everyone pulls, we can add that file to git ignore and everyone can have personal control over the room IDs

As far as I know, there's no automatic way to grab room IDs from Discord. They have to be manually copied and pasted from right-clicking on the channel names.